hack/lib.sh

```diff
@@ -32,6 +32,15 @@ set -o monitor
 # fi
 OS="$(echo $(uname) | tr '[:upper:]' '[:lower:]')"
 
+# Set the container runtime to docker or nerdctl.
+if command -v docker &> /dev/null; then
+  CONTAINER_RUNTIME=docker
+elif command -v nerdctl &> /dev/null; then
+  CONTAINER_RUNTIME=nerdctl
+else
+  CONTAINER_RUNTIME=docker
+fi
+
```

Comment:
I think it would be better if this was

```suggestion
CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
```

That way, we default to docker, but users can select whatever container runtime they want to use with setting `CONTAINER_RUNTIME` as environment variable. If people dont use either docker or nerdctl, they are still stuck here.

Reply:
changed to support user input, however, provided runtime must be compatible with docker.